htdocs/admin/content/galaxy/edit.php

```diff
@@ -1,624 +1,573 @@
 <?PHP
 
 use EtoA\Core\Configuration\ConfigurationService;
+use EtoA\Universe\PlanetRepository;
+use EtoA\Universe\AsteroidsRepository;
+use EtoA\Universe\EmptySpaceRepository;
+use EtoA\Universe\EntityRepository;
+use EtoA\Universe\EntityType;
+use EtoA\Universe\NebulaRepository;
+use EtoA\Universe\StarRepository;
+use EtoA\Universe\WormholeRepository;
+use EtoA\User\UserRepository;
+use Symfony\Component\HttpFoundation\Request;
 
 /** @var ConfigurationService */
 $config = $app['etoa.config.service'];
 
-$id = $_GET['id'];
-if (isset($id) && $id > 0)
+/** @var EntityRepository */
+$entityRepo = $app['etoa.universe.entity.repository'];
```

Comment:
I'm not really sure but in other places you used `$app[\EtoA\Universe\PlanetRepository::class];` to get the PlanetRepository for example. Is there a reason you're using magic strings here?

Reply:
That was me starting it and then I realized that using the class name is way easier ...

Reply:
I prefer to use `use` statements, see the head of the file. It is more convenient, as I can use the auto complete / intelli sense of my IDE to automatically add the use statements.

Reply:
What he is referring to is that instead of using `'etoa.universe.entity.repository'` we can use `EntityRepository::class` as identifier in $app

Reply:
Aaaah. Yes, that would also make sense, was not sure if this is supported.